cmd/v/internal/help/build-c.txt

```diff
@@ -0,0 +1,68 @@
+Usage: v [build flags] ['build'] <file.v|directory>
+
+This command compiles the given target, along with their dependencies, into an executable.
+
+This help topic explores C-backend specific build flags.
+For help regarding building an executable, see `v help build`.
+
+These build flags are enabled on `build` and `run` as long as the backend is set to `c`:
+
+   -arch <architecture>
+      Changes the architecture that V will tell the C compiler to build.
+      List of supported architectures: `x86` and `x64` (default).
+
+   -cc <compiler>, -compiler <compiler>
+      Changes the C compiler V invokes to the specified compiler.
+      The C compiler is required to support C99.
+      Officially supported/tested C compilers include: `clang`, `gcc`, `tcc`, `mingw-w64` and `msvc`.
+
+   -cf <flag>, -cflags <flag>
+      Passes the provided flag as is to the C compiler.
+      Can be specified multiple times to provide multiple flags.
+      Use quotes to wrap the flag argument if it contains spaces.
+
+   -cg, -cdebug
+      Enables debug mode while preserving C line numbers in the compiled executable.
+      This allows issues regarding C compilation to be located more easily.
+
+   -compress
+      Strips the compiled executable to compress it.
+
+   -csource <prettify|keep|drop>
+      Specifies how V deals with the intermediate C source code.
+      * `prettify`       - The C source code will be kept.
+                           `clang-format` is used to prettify the C source code before compiling it.
+      * `keep`           - The C source code will be kept as generated by V.
+      * `drop` (default) - The C source code will be deleted after compiling the executable.
+
+   -freestanding
+      Builds the executable without dependency on libc.
+      Supported only on `linux` targets currently.
+
+   -live
+      Builds the executable with live capabilities (`[live]`).
+
+   -manual-free
+      Enables manually freeing on all V code. This will cause memory leaks in vlib.
+      Currently enabled until a stable auto-free system is in place.
+
+   -os <os>, -target-os <os>
+      Changes the target OS that V tries to compile for.
+      By default, the target OS is the host system.
+      When OS is `cross`, V will attempt to output cross-platform C code.
+      List of OS supported by V: `linux`, `windows`, `mac`, `freebsd`, `openbsd`, `netbsd`,
+                                 `dragonfly`, `solaris`, `android` and `haiku`.
+
+   -sanitize
+      Passes flags related to sanitization to the C compiler.
+
+   -shared
+      Tells V to compile a shared object instead of an executable.
+      The resulting file extension will be `.dll` on Windows and `.so` on Unix systems
+
+Notable internal flags (should not be used as it is subject to change):
+   -repl
+      Builds the provided code in REPL mode.
+
+   -solive
```

Comment:
I am not sure that there is a need to document this here. It is not intended to be seen/used by end users.